bindings/python/pymongoarrow/libbson.pxd

```diff
@@ -38,7 +38,8 @@ cdef extern from "<bson/bson.h>":
         pass
 
     ctypedef struct bson_decimal128_t:
-        pass
+        uint64_t high
+        uint64_t low
```

Comment:
Do we need to reorder these fields for big endian machines like libbson does?

Reply:
Handled in `lib.pyx`

Reply:
Is it possible to reorder the struct fields themselves like what's done in libbson?

Reply:
I removed support for `Decimal128` on big-endian systems.